### PR TITLE
Update actions with start and stop locations at PVL lab

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description
Update V2X Hub Port Drayage Plugin actions with new start/stop locations for testing in PVL Lab:
- PICKUP location: [3.7, -5.8]
- EXIT_STAGING_AREA location: [0.0, 0.2]

Latitude is mapped to Y axis, and longitude is mapped to X axis.
The locations are picked from the pvl graph: https://github.com/usdot-fhwa-stol/c1t_bringup/blob/develop/graphs/pvl_graph.geojson?short_path=4609dc2

## Related Issue
https://usdot-carma.atlassian.net/browse/CF-495

## Motivation and Context
NA

## How Has This Been Tested?
NA

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by paulbourelly999 — CHANGES_REQUESTED** [review]
Please add a related JIRA Ticket. Also It may be worth while creating a separate SQL file for the C1T testing. This file is currently used for our functionality testing. If we change things here that impact that we need to update that documentation

**Review by paulbourelly999 — CHANGES_REQUESTED** [review]
I know I was initially on the fence about this but could we add a README here outlining what this sql file is for and how to use it.
Example
### PVL Actions
These are Port Drayage actions created for the PVL Lab for testing of C1T functionality.
**Optional: Included some screen shot of the actions on a map**
### Instructions 
```docker-compose
db:
   image: mysql:8.0
   volumes:
       ./mysql/pvl_lab/port_drayage.sql


```

**Comment by dan-du-car** [comment]
> Please add a related JIRA Ticket. Also It may be worth while creating a separate SQL file for the C1T testing. This file is currently used for our functionality testing. If we change things here that impact that we need to update that documentation

@paulbourelly999  Addressed

File: configuration/mysql/pvl_lab/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# PVL Actions
+These are Port Drayage actions created for the PVL Lab for testing of C1T functionality.
+
+## Instructions
```
Comment (paulbourelly999) [inline configuration/mysql/pvl_lab/README.md:4]: @dan-du-car  lol these were place holder instructions. I think as the are currently written they do not work. You have to update the volume here and maybe add a sentence about replacing the old actions with the ones configured for PVL

**Review by paulbourelly999 — CHANGES_REQUESTED** [review]
My instructions were just place holders. They need a little editing to make them actually work.

**Pull request merged**